homeassistant/components/spotify/media_player.py

```diff
@@ -104,6 +105,7 @@ def __init__(self, session, spotify: Spotify, me: dict, user_id: str, name: str)
         self._devices: Optional[List[dict]] = []
         self._playlist: Optional[dict] = None
         self._spotify: Spotify = None
+        self._start_random = False
```

Comment:
@frenck it is not possible to use the `shuffle` property because spotify does not allow u to enable shuffle when you are not playing anything. Therefore if nothing is playing yet and you want to start a playlist with a random song you cannot use the `shuffle` property, but need an assumed state for the shuffle property.

Or do you know another way to do this @frenck

Reply:
That statement you make is incorrect:

![2020-02-18 11 40 24](https://user-images.githubusercontent.com/195327/74728696-84a65080-5243-11ea-9aef-e62699a18b81.gif)

Reply:
@frenck cool did not know that that worked.
I will change it and test it tonight at my system.